crates/hc_sandbox/src/cli.rs

```diff
@@ -98,6 +98,12 @@ pub enum HcSandboxSubcommand {
     /// Make a call to a conductor's admin interface.
     Call(crate::calls::Call),
 
+    /// Authorize credentials for making zome calls.
```

Comment:
```suggestion
    /// Create and authorize credentials for making zome calls.
```

Reply:
If I understand it correctly this creates and authorizes credentials, i.e. it's not possible to authorize existing credentials, right?

Reply:
Correct yes, if the API permitted it then I would have added an operation to revoke the credentials when you're done. IMO credentials should be used by a single client and when you're done with what you were using them for, they should be removed to avoid the risk of somebody else using them. It's not okay that these credentials are valid over the network and can't be removed again.

Reply:
Opened an issue about this too https://github.com/holochain/holochain/issues/4596